packages/components/src/components/Divider/ContentfulDivider.tsx

```diff
@@ -1,22 +1,14 @@
 import React from 'react';
 import './ContentfulDivider.css';
-import { css, cx } from 'emotion';
-
-const styles = {
-  hr: css({
-    border: 'none',
-  }),
-};
```

Comment:
The components package doesn't have `emotion` installed. I moved this style to the component's css file instead.